lib/class/api.class.php

```diff
@@ -1481,7 +1481,7 @@ public static function playlist_songs($input)
      */
     public static function playlist_create($input)
     {
-        if (!self::check_parameter($input, array('name', 'type'), 'playlist_create')) {
```

Comment:
I guess, this is ok because type is optional according to the documentation.

Reply:
But: If the type-Key is not defined in the input-array, Line 1488 will raise a warning because of the missing key.

Possible solution:
```
$type = $input['type'] ?? 'private';
```

Reply:
use isset and that's fine with me
$type = (isset($input['type'])) ? $input['type'] : 'private';

Reply:
@lachlan-00 just fyi, `??` is the null coalescing operator (>= php7) which checks if an expression evaluates to null and not the short form of the ternary operator (`?:`).

Reply:
just keeping it consistent ;)